### ISSUE TITLE
textWidth() causes a NullPointerException for P2D and P3D unless a font is explicitly specified

### ISSUE THREAD

**Issue by MinteZ** [processing/processing#5137]
```java
void setup() {
    size(100, 100, P2D); 
    //textFont(createFont("Consolas", 12)); 
    println(textWidth('c'));
}
```

This code seems to be the culprit (line 3485 of PGraphicsOpenGL):

`Object font = this.textFont.getNative();`

**Comment by GKFX** [comment]
@MinteZ I've posted a fix. As a temporary workaround you can call `textAscent();` beforehand as that method calls `defaultFontOrDeath` properly.

**Comment by MinteZ** [comment]
@GKFX Alright. Thank you.

**Comment by benfry** [comment]
Fix incorporated for 3.3.5.

**Comment by MinteZ** [comment]
@benfry Good to hear. Thank you and @GKFX!
